src/entities/twin/libs/helpers.ts

```diff
@@ -12,14 +12,31 @@ const stringLikeFilters = [FilterFields.twinNameLikeList];
 
 const arrayLikeFilters = [
   FilterFields.twinIdList,
+  FilterFields.twinClassIdList,
   FilterFields.statusIdList,
+  FilterFields.createdByUserIdList,
   FilterFields.assignerUserIdList,
-  FilterFields.twinClassIdList,
+  FilterFields.headTwinIdList,
+  FilterFields.tagDataListOptionIdList,
+  FilterFields.markerDataListOptionIdList,
 ];
 
 // List of filter keys for choice-mapping
 const choiceMappingFilters: any = [];
 
+const toArrayOfString = <T extends { [key: string]: any }>(
```

Comment:
remove this function. its redundant

Instead, use `toArrayOfString` from `@/shared/libs/helpers`

ℹ️ [here](https://github.com/alcosi/twinfaces/blob/TWINFACES-69-permission-resource-link/src/shared/libs/helpers.ts) you can find version of `toArrayOfString` function which supports passing `key` as an arguments